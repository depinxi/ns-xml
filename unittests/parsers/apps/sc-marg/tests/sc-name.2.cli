-c abc def ghi --switch-a sub-command "This should succeed"