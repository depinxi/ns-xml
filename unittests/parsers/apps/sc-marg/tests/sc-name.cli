-c abc def ghi -a sub-command "There is only global option names and option args before sc name"