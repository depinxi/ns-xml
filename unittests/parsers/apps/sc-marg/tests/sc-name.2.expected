CLI: "-c", "abc", "def", "ghi", "--switch-a", "sub-command", "This should succeed"
Value count: 1
Values: "This should succeed"
Error count: 0
Subcommand: sub-command
switchA=True
argumentB=
multiargC=abc def ghi
sub-command_argumentD=
sub-command_switchE=False
sub-command_switchA2=False
